agents_infra/agents_infra/main.py

```diff
@@ -0,0 +1,62 @@
+import json
+import logging
+import os
+import sys
+
+from agents_infra.agents import agent_factory
+from agents_infra.supervisor import AgentSupervisor
+from agents_infra.tasks import periodic_task_wrapper, task_factory
+from agents_infra.utils.logtools import maybe_log_message
+
+
+def load_tasks_config():
+    config_paths = [
+        os.path.join(os.path.dirname(__file__), 'tasks_config.json'),
+        os.path.join(os.path.dirname(__file__), 'tasks_config.json.template'),
```

Comment:
I understand that `task_config.json.tamplate` is provided rather for us - so that we know what config pattern to follow. Why is it hardcoded here?